CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 # CMakeLists in this exact order for cmake to work correctly
 cmake_minimum_required(VERSION 3.5)
 
+set(LANGUAGE, C)
```

Comment:
Do we need specify the language here? I would like to have some copied files from the main printer repo to make sure they are 1:1 and I would like to write them in c++.

Reply:
Fixed in 6945f3a06adbb5a5fbb37f87395c202c4c687915